pagopa/io-app
# feat: [IOCOM-347] Handling of events emitted by the FIMS WebView

## ⚠️ This PR depends on #4744 ⚠️
Please review first #4744 in order to avoid duplicate review work.

## Short description
This PR updates the handling of events emitted by the FIMS WebView.
Currently the handled events are:
- `CLOSE_MODAL`: closes the WebView
- `SHOW_ALERT`: shows an alert
- `SHOW_TOAST`: shows a toast
- `SET_TITLE`: sets the title of the screen

## List of changes proposed in this pull request
- removed unused events
- removed unused components
- added the possibility to show a toast from the WebView
- updated tests

## How to test
Using the `io-dev-api-server`, change the `withCTA` configuration to generate a message with FIMS CTA and enable the [FIMS dev feature flag](https://github.com/pagopa/io-dev-api-server/blob/e3ac6784acecbb5ad80a44dfc7777f80946dcab5/src/payloads/backend.ts#L45) in `src/payloads/backend.ts`.
Select such message and tap the FIMS CTA at the bottom.
Emit the events from the WebView and check that they are handled correctly.
